.travis.yml

```diff
@@ -1,20 +1,15 @@
 language: cpp
 dist: xenial
 
-script:
-  - ./autogen.sh
-  - test -z $BUILD_DIR || { mkdir -p $BUILD_DIR; cd $BUILD_DIR; }
-  - ${SRC_DIR:-.}/configure $CONFIGURE
-  - test "$NO_COMPILE" || make
-  - make ${TEST:-test}
-
-matrix:
+script: ci/build.sh
+
+jobs:
   include:
 
-  # Job 1: Build (in separate build directory) on Linux with native GCC
+  # Job 1: Build on Linux with native GCC
```

Comment:
Since CMake supports in-place builds (`cmake . && make ...`) I think that it would make sense to let at least one of the jobs test that it works since some end users surely will use that method.